vis/js/templates/ZoomedOutHeading.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const ZoomedOutHeading = ({
+  introIcon,
+  introLabel,
+  children: title,
+  additionalFeatures,
+}) => {
+  return (
+    // html template starts here
+    <h4>
+      {title}{" "}
+      <a data-toggle="modal" data-type="text" href="#info_modal" id="infolink">
+        <span
+          id="whatsthis"
+          dangerouslySetInnerHTML={{ __html: introIcon }}
```

Comment:
can we find a way to avoid this? Maybe by making an icon component?

Reply:
Good question... the icon comes from config and it's a fontawesome unicode entity, e.g. `&#xf05a;`. I could create an icon component instead (or even better, directly use fontawesome library for react), but then it wouldn't be possible to change the icon from config so simply - a change in code would be needed.

Let's talk about this at the meeting.